deploy/health_check/Pipfile

```diff
@@ -0,0 +1,22 @@
+[[source]]
+
+verify_ssl = true
+url = "https://pypi.python.org/simple"
+name = "pypi"
+
+
+[dev-packages]
+
+
+
+[packages]
+
+requests = "*"
+flask = "*"
+flask-cors = "*"
+gunicorn = "*"
```

Comment:
always set specific versions (probably the latest ones) to avoid unexpected behavior

Reply:
btw the structure of this file looks like it was edited manually - was it?

Reply:
Not edited manually, and the version lock is in the Pipfile.lock file (pipenv does that manually)